src/OrbitGl/GetThreadColor.h

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
```

Comment:
class? You mean file? I don't agree. We can rename it when the scope changes, but we shouldn't prepare for a refactoring that might not happen in the next year.

Reply:
Should I increase this scope of this bug to include refactoring all color definitions and putting them in this class or should I make it, its own separate bug?

Reply:
Yes, the file. It makes sense. Anyway I would call it `ThreadColor` instead of `GetThreadColor`

Anyway, @Mahmood-Darwish I think you have two options.

1- Either only having Thread's color logic here: With this I mean moving back the method that takes an string as a parameter (used in AsyncTrack and GpuMarkerSubTrack)
2- Or renamed to something more Track generic (`TrackColor`??) where we have methods for `GetThreadColor` and also `GetTrackColor`

Reply:
> Should I increase this scope of this bug to include refactoring all color definitions and putting them in this class or should I make it, its own separate bug?

I would choose between only `ThreadColor` or `TrackColors`. Color in general in OrbitGl could be around 400 changes???

Reply:
I decided to make it responsible for the thread colors only, are there any other places where thread colors are used and I should modify them?

Reply:
I don't think so